examples/webgpu_custom_fog_background.html

```diff
@@ -116,7 +112,6 @@
 				controls.maxDistance = 5;
 				controls.target.set( 0, - 0.1, - 0.2 );
 				controls.update();
-				controls.addEventListener( 'change', render ); // use if there is no animation loop
```

Comment:
Why do you prefer to insert animation loops when they are not necessary?

Reply:
Simply because the examples are easier to debug with an animation loop.

Reply:
>Because the examples are easier to debug with an animation loop. And it simplifies the code.

I don't think that is a very good reason, IMHO.

>I've also made the experience that the E2E screenshots work more reliably if animation loops are in place.

Nor that one.